myst_parser/__init__.py

```diff
@@ -4,9 +4,11 @@
 def setup(app):
     """Initialize Sphinx extension."""
     from myst_parser.sphinx_parser import MystParser
+    from myst_parser.myst_refs import MystReferenceReslover
```

Comment:
shouldn't this be `Resolver` instead of `Reslover` (and throughout)?

Reply:
oops yeh fixed